test/types/objectSearch.test.tsx

```diff
@@ -0,0 +1,68 @@
+import faker from 'faker';
+
+import { Tester } from '@mighty-justice/tester';
+
+import { fillInFieldConfig } from '../../src';
+import ObjectSearch from '../../src/inputs/ObjectSearch';
+
+function getDefaults (overrides?: any) {
```

Comment:
Part of me wants this `getDefaults` to be a subset of / used in [this one](https://github.com/mighty-justice/fields-ant/blob/a8802f0f2325fd15105136fdfde6ea48b5f90498/test/types/objectSearchCreate.test.tsx#L8). What are your thoughts?

Reply:
I actually prefer some copy-pasting in tests so that they work more independently.